src/db.rs

```diff
@@ -344,4 +344,5 @@ CREATE UNIQUE INDEX IF NOT EXISTS review_prefs_user_id ON review_prefs(user_id);
     "
 ALTER TABLE review_prefs ADD COLUMN IF NOT EXISTS max_assigned_prs INTEGER DEFAULT NULL;
 ",
+    "ALTER TABLE review_prefs DROP COLUMN assigned_prs;",
```

Comment:
We can also just skip this migration and keep the field in the DB, but I don't think that it's needed anymore.

Reply:
Ok, fair enough, I'll keep the column.